tests/components/cover/test_rflink.py

```diff
@@ -416,33 +416,6 @@ def listener(event):
     assert hass.states.get(DOMAIN + '.test').state == STATE_OPEN
 
 
-async def test_disable_automatic_add(hass, monkeypatch):
```

Comment:
automatic_add is not supported for RFLink cover so remove the test.
@javicalle Maybe you can confirm it's correct to remove the test case?

Reply:
I think you're right
I have reviewed the `cover` component and it is as you say. 
Sorry for the inconvenience.